src/components/header.js

```diff
@@ -1,5 +1,113 @@
-import React from 'react'
+import React, { useState } from 'react'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import Typography from '@material-ui/core/Typography'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import {
+  makeStyles,
+  createMuiTheme,
+  ThemeProvider,
+  Avatar,
+  Grid,
+} from '@material-ui/core'
 
-const Header = () => <div>Header</div>
+const transparentTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#E5E5E5', contrastText: '#ffffff' },
+  },
+})
 
-export default Header
+const blackTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#000000' },
+  },
+})
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  header: {
+    margin: 'auto',
+    height: 60,
+    color: 'ffffff',
+  },
+  onScroll: {
+    height: 55,
+  },
+  white: {
+    color: '#ffffff',
+    backgroundColor: '#ffffff',
+  },
+  headerItems: {
+    color: '#111111',
+  },
+}))
+
+export default function Header(props) {
+  const classes = useStyles()
+  const [scrollValue, setScrollValue] = useState(0)
+
+  window.onscroll = logScroll
+
+  function logScroll(e) {
+    const { scrollY } = { ...e.currentTarget }
+    setScrollValue(scrollY)
+  }
+
+  return (
+    <React.Fragment>
```

Comment:
You need not use `<React.Fragment>` here the return statement is returning single JSX element.